openstack/python/tar/slipstream_openstack/OpenStackClientCloud.py

```diff
@@ -90,6 +88,12 @@ def _finalization(self, user_info):
         except Exception:
             pass
 
+    def _get_iaas_attributes(self):
+        self.flavors = self._thread_local.driver.list_sizes()
```

Comment:
```python
self.flavors = self._thread_local.driver.list_sizes()
```
cannot be set conditionally because it's used for the describe-instances at [line 255](https://github.com/slipstream/SlipStreamConnectors/pull/78/files#diff-7c47d1faa03b4c03eddd7d9ff2deb126R255).